packages/manager/modules/overthebox/src/overthebox/actions/overTheBox-actions.controller.js

```diff
@@ -0,0 +1,61 @@
+export default /* @ngInject */ function(
+  $translate,
+  $q,
+  $stateParams,
+  PAGINATION_PER_PAGE,
+  OvhApiOverTheBoxDevice,
+  TucToast,
```

Comment:
`telecom-universe-components` and `ovh-api-services` should be added as dependency to your module (`ovhManagerOtbActions`)